docs/_static/vcr/v2.yaml

```diff
@@ -0,0 +1,281 @@
+          in: query
+          description: the did of the issuer
+          example: did:nuts:456
+          required: false
+          schema:
+            type: string
+      tags:
+        - credential
+      responses:
+        "200":
+          description: Credential has been found and is returned.
+          content:
+            application/vc+json:
+              schema:
+                $ref: '#/components/schemas/ResolutionResult'
+        default:
+          $ref: '../common/error_response.yaml'
+  /internal/vcr/v2/issuer/vc/{id}:
+    parameters:
+      - name: id
+        in: path
+        description: URL encoded ID.
+        required: true
+        example:
+          - "did:nuts:B8PUHs2AUHbFF1xLLK4eZjgErEcMXHxs68FteY7NDtCY#90382475609238467"
+        schema:
+          type: string
+    delete:
+      summary: "Revoke an issued credential"
+      description: |
+        Revoke a credential.
+
+        error returns:
+        * 400 - Credential can't be revoked. Most likely due to a missing private key.
+        * 404 - Credential is not found
+        * 409 - Credential has already been revoked
+        * 500 - An error occurred while processing the request
+      operationId: "revokeVC"
+      tags:
+        - credential
+      responses:
+        "200":
+          description: Revocation has been processed locally. It has also been published to the network.
+          content:
+            application/vc+json;type=revocation:
+              schema:
+                $ref: '#/components/schemas/Revocation'
+        default:
+          $ref: '../common/error_response.yaml'
+  /internal/vcr/v2/verifier/vc:
+    post:
+      summary: Verifies a Verifiable Credential
+      description: |
+        Verifies a Verifiable Credential: It checks the signature, expiration, revocation status and if the issuer is trusted.
+        
+        error returns:
+        * 400 - One or more of the given parameters are invalid
+        * 500 - An error occurred while processing the request
+      operationId: "verifyVC"
+      tags:
+        - credential
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/VerifiableCredential'
+      responses:
+        "200":
+          description: "The verification result"
+          content:
+            application/vc+json:
+              schema:
+                $ref: '#/components/schemas/VCVerificationResult'
+        default:
+          $ref: '../common/error_response.yaml'
+
+
+components:
+  schemas:
+    IssueVCRequest:
+      type: object
+      description: A request for issuing a new Verifiable Credential.
+      required:
+        - type
+        - issuer
+        - credentialSubject
+      properties:
+        "@context":
+          description: |
+            The resolvable context of the credentialSubject as URI. If omitted, the "https://nuts.nl/credentials/v1" context is used.
```

Comment:
What if you add an unknown VC type with an external resolvable context?

Reply:
Currently nothing, but when we add support for JDON-LD, this context must be resolved. We can add a flag to the config to allow for resolving contexts.